Mathlib/Data/Rat/Init.lean

```diff
@@ -1,18 +1,57 @@
 /-
 Copyright (c) 2019 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl, Mario Carneiro
+Authors: Johannes Hölzl, Mario Carneiro, Yaël Dillies
 -/
+import Mathlib.Init.Data.Nat.Notation
 import Mathlib.Mathport.Rename
-import Std.Data.Rat.Basic
+import Mathlib.Tactic.Lemma
+import Mathlib.Tactic.ProjectionNotation
+import Mathlib.Tactic.TypeStar
+import Std.Classes.RatCast
 
 #align_import data.rat.init from "leanprover-community/mathlib"@"f340f229b1f461aa1c8ee11e0a172d0a3b301a4a"
 
 /-!
-# Notation for the rational numbers -/
+# Definition of the rational numbers
```

Comment:
Well... It's not really defined in this file...
Is this title strictly better than the old one?